src/Ingredient.css

```diff
@@ -0,0 +1,12 @@
+.selected {
```

Comment:
Generally, I would encourage avoiding `px` and use `rems` or `ems` instead. `px` are bad for responsive design and accessibility.

Reply:
Also - I would err on the side of idiomatically organizing your css declarations as you go - just to avoid forgetting at the last moment!

Reply:
We are going to make major updates to our CSS this week, will make these changes with that refactoring and addition of Sass.